vms/platformvm/vm.go

```diff
@@ -90,6 +92,10 @@ type VM struct {
 	manager   blockexecutor.Manager
 }
 
+type Config struct {
+	ChecksumsEnabled bool `json:"checksums-enabled"`
+}
```

Comment:
I felt like now was as good of a time as any to introduce `P-chain` specific configs. We'll probably use this more in the future for APIs and other VM level configs